web-ui/src/main/resources/catalog/components/edit/onlinesrc/partials/remote-record-selector.html

```diff
@@ -28,13 +28,18 @@
     </div>
   </form>
 
-  <div class="list-group" data-ng-if="remoteRecord.title">
+  <div class="list-group" data-ng-if="remoteRecord.uuid">
```

Comment:
```suggestion
  <div class="col-md-offset-2 list-group" data-ng-if="remoteRecord.uuid">
```

to keep it aligned with the URL ? 

![image](https://github.com/geonetwork/core-geonetwork/assets/1701393/2073c419-ea20-4e44-9ede-74c0fbfeda2a)